src/transformers/utils/import_utils.py

```diff
@@ -246,6 +246,12 @@
 except importlib_metadata.PackageNotFoundError:
     _pyctcdecode_available = False
 
+_kenlm_available = importlib.util.find_spec("kenlm") is not None
+try:
+    _kenlm_version = importlib_metadata.version("kenlm")
+    logger.debug(f"Successfully imported kenlm version {_kenlm_version}")
+except importlib_metadata.PackageNotFoundError:
+    _kenlm_available = False
```

Comment:
We don't need the kenlm version anywhere right? I'd just test for `importlib.util.find_spec("kenlm") is not None` in the util function down there.
```suggestion
```

Reply:
Thanks !